azure-pipelines-integration.yml

```diff
@@ -62,7 +62,7 @@ parameters:
 - name: queueName
   displayName: Queue Name
   type: string
-  default: windows.vs2022preview.amd64.open
+  default: windows.vs2022preview.scout.amd64.open
```

Comment:
Don't you need to move `azure-pipelines-integration-corehost.yml` as well?

Reply:
Ah thanks reminds me this